test/heatmap-calendar.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<!--
```

Comment:
There is already a `calendar-heatmap` case. Add a new `heatmap-calendar` may be confusing

Reply:
I've already deleted it